src/test/java/io/reactivex/internal/operators/observable/ObservableRefCountTest.java

```diff
@@ -1345,4 +1345,19 @@ public void timeoutDisposesSource() {
 
         assertTrue(((Disposable)o.source).isDisposed());
     }
+
+    @Test
+    public void disconnectBeforeConnect() {
+        BehaviorSubject<Integer> subject = BehaviorSubject.create();
+
+        Observable<Integer> observable = subject
+                .replay(1)
+                .refCount();
+
+        observable.takeUntil(Observable.just(1)).test();
```

Comment:
For consistency, this line either needs same comment as flowable test, or it should be removed from flowable test

Reply:
Removed from the other place.